app/Http/Controllers/BackupController.php

```
@@ -81,6 +81,28 @@ public function run(Server $server, Backup $backup): RedirectResponse
             ->with('info', 'Backup is being created...');
     }
 
+    #[Post('/{backup}/enable', name: 'backups.enable')]
+    public function enable(Server $server, Backup $backup): RedirectResponse
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$server'. (undefined)

(UnusedFormalParameter)